tests/unit/amo/pages/TestAddonReviewList.js

```diff
@@ -1142,15 +1142,15 @@ describe(__filename, () => {
         location: createFakeLocation({ query: { score } }),
       });
 
-      expect(selector).toHaveValue(score);
+      expect(selector).toHaveProp('value', score);
```

Comment:
The changes in this file are required because the `toHaveValue` assertion from `@testing-library/jest-dom/extend-expect` is being picked up, instead of from `jest-enzyme`, now that we are loading the former library second. This is the behaviour we want, but it can cause problems for tests still using Enzyme. Luckily, this seems to be the only test affected.